src/containers/sidebar/sidebar-item/sidebar-item.js

```diff
@@ -33,7 +36,7 @@ const SideBarItem = ({ category, handlerItem, models, translationsKey, mainItemS
           {models.map((model) => (
             <ListItem button className={styles.nested} key={model._id} onClick={handlerItem}>
               <Link
-                to={`/catalog/:category?${sort}=${POPULARITY}&${page}=${defaultPage}&${categoryFilter}=%2C${category}&${modelsFilter}=%2C${model._id}`}
+                to={`${pathToCategory}?${sort}=${POPULARITY}&${page}=${defaultPage}&${categoryFilter}=%2C${category}&${modelsFilter}=%2C${model._id}&${countPerPage}=${defaultCount}`}
```

Comment:
It can move to variable and after that put here. As you did it in slider-home-page.js file with pathToCatalog.
And also if we don't set countPerPage in url, it will add by default.

Reply:
fixed